content/en/docs/Releases/_index.md

```diff
@@ -0,0 +1,590 @@
+---
+title: Releases 
+linkTitle: "Releases"
+weight: 120
```

Comment:
Can you change the weight to 130? Each weight should be higher than the previous page, otherwise docsy will sort in lexical order.